src/main/java/sim/coffee/Logger.java

```diff
@@ -10,7 +10,10 @@
 	public enum OrderState {
 		ENTER,
 		EXIT,
-		PROCESSED;
+		PROCESSED,
+		ENTERKITCHEN,
+		EXITKITCHEN,
+		SERVED;
```

Comment:
Feels like quite a few redundant moves here. Since orders are all kinda a one way thing. 

Feels like could be cut down to just enter, processed, enterKitchen and Served. More like when a food is served, it also means that its no longer in the kitchen, etc.

Reply:
As I was saying to Kyle, the impression I got from the coursework requirements was that they want to see the specific events recorded in the log as they're taking place, hence the many enums.

Reply:
I think all the events currently in are worthwhile